### PR TITLE
Issue 7118: Add reply WireCommands to DEBUG log.

### LINKED ISSUE #7118: Add reply WireCommands to DEBUG logs
**Is your feature request related to a problem? Please describe.**
Currently we only log the requests in ServerConnectionInboundHandler but do not log corresponding replies.

**Describe the solution you'd like**
Add debug logs in TrackedConnection.send

**Describe alternatives you've considered**
There is no other alternative. 

**Additional context**
None

### PR DESCRIPTION
**Pull request by sachin-j-joshi**
**Purpose of the change**  
Fixes #7118 

**What the code does**  
Add reply WireCommands to DEBUG log.

**How to verify it**  
All tests should pass. Manually check logs.

### CODE REVIEW

**Review by ShwethaSNayak — APPROVED** [review]
LGTM

**Review by krishan-Rai — APPROVED** [review]
LGTM

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/TrackedConnection.java [hunk 1]
```diff
@@ -66,6 +68,7 @@ void adjustOutstandingBytes(int delta) {
      */
     void send(WireCommand cmd) {
         this.connection.send(cmd);
+        log.debug("Sent response: {}", cmd);
```
Comment (abhinb) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/TrackedConnection.java:71]: wouldnt logging it here result in a flood of logs for the append path/connection hello's etc.

Comment (tkaitchuck) [reply segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/TrackedConnection.java:71]: It's debug. So that's fine.

**Review by tkaitchuck — APPROVED** [review]

**Review by abhinb — APPROVED** [review]

**Pull request merged**